lib/features/reps_core/reps_core.info

```diff
@@ -2,21 +2,20 @@ name = REPS core
 description = The core feature fo the REPS websites
 core = 7.x
 package = REPS
-php = 5.6
-dependencies[] = bean
 dependencies[] = ckeditor_tabber
-dependencies[] = ds
 dependencies[] = ds_extras
 dependencies[] = ec_embedded_video
 dependencies[] = ecas_env
+dependencies[] = entity
 dependencies[] = entityreference
 dependencies[] = extlink
 dependencies[] = fe_block
 dependencies[] = fe_nodequeue
-dependencies[] = feeds
 dependencies[] = feeds_tamper
```

Comment:
feeds has been removed from dependency but not feeds_tamper and feeds_xpathparser. Is that normal?